notifications.bs

```diff
@@ -84,7 +84,8 @@ an existing notification.
 
 <p>A <a>notification</a> has an associated
 <dfn for=notification id=silent-preference-flag>silent preference</dfn> boolean, which is initially
```

Comment:
null or a boolean (might need some general rewording to make that more palatable)